autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -42,12 +42,16 @@ export function removeCourseCard(index: number): RemoveCourseAction {
    * Helper action creator that generates plain old UpdateCourseActions
    * @param index the index of the course card to update in the CourseCardArray
    * @param courseCard the options to update
+   * @param canCreate if this update action is allowed to create a new course card
    */
-function updateCourseCardSync(index: number, courseCard: CourseCardOptions): UpdateCourseAction {
+function updateCourseCardSync(
+  index: number, courseCard: CourseCardOptions, canCreate = false,
+): UpdateCourseAction {
   return {
     type: UPDATE_COURSE_CARD,
     index,
     courseCard,
+    canCreate,
```

Comment:
What's the use of `canCreate`? I don't see it used anywhere

Reply:
Oops I rebased and forgot to push the change, that was my original solution but did something else (see my comment on addCourseCard)